### ISSUE TITLE
File-based unit and subunit editing

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/meme-2023#60]

**Comment by benloh** [comment]
See also a second [slack discussion](https://inquirium.slack.com/archives/C04DTUVARMM/p1731622951260889)
> when we are running studies in more than 1 classroom we want to be able to somehow create one then make copies otherwise it’s a pain in the butt. For Net.Create it’s easy because we make a template and db and then just copy and rename the template.

**Comment by benloh** [comment]
# Design

## Unit Management

- Link `unit` to `project_id`
- Once you've selected a unit for project, you cannot change the unit.

Components 
- `WUnitList` -- display list of available units
- `WNewProjectDialog` -- click "New Project", open new dialog that displays list of units to select.


## File Structure
```
/units
  /wetlands.            # folder name works as id?
    wetlands.yaml 
    /resources
      fish.pdf
      marsh.pdf
  /beavers
    beavers.yaml
    /resources
      beaver.pdf
      trees.pdf
      dam.pdf      
```

**Comment by benloh** [comment]
# TO DO

## Ratings
- [x] `class-ratings.js`
- [x] `WRatingsDialog.jsx` -- `DoOpen` calls `ADM.GetRatingsDefinition(classroomId)`
   - [x] Convert this to a `class-ratings.GetRatingsByUnit` call?
- [X] `WRatingsList` -- relies on `RATINGS.getDefinitions()` from `class-ratings` for data
- [X] `WRatingButton` -- relies on `RATINGS.GetLabel()` from `class-ratings` for data
- [X] `WRatingIcon` -- no dependencies

## Criteria
- [x] Review criteria data updates?
- [x] Remove "Criteria" Admin UI
- [x] Remove "Criteria for a Good Model" from Help

## Resources
- [x] `adm-data.LoadModel` loads resources from unit

## Comment Types
- [x] Load from Unit

## Cleanup
- [x] Admin Page: 
   - [x] Remove "Ratings Definitions"
   - [x] Remove "Criteria for a Good Model"
   - [x] Remove "Resources" from admin page

**Comment by benloh** [comment]
@jdanish Currently "Criteria for a Good Model" is only defined on the Help dialog.  I believe it used to be used with the old sticky not system.  I don't believe it's used anywhere else, since sticky notes were replaced by comments and comment types.  Are criteria still important for Units?  Should we get rid of them and just rely on comment types to define criteria?  Or did you still want to be able to define criteria in the Help dialog?

**Comment by jdanish** [comment]
Correct. We can remove it and use it just in the comments since no one went to help anyhow for that. So, yes, all in comments.
